src/app/shared/editor/services/monaco.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from "@angular/core";
+import * as monaco from "monaco-editor";
+import { MaybeNull } from "../../../common/app.types";
+/* eslint-disable */
```

Comment:
Remove this full fill pragma and fix the warnings.

Reply:
Removed and fixed